addons/mail/static/tests/tours/discuss_public_tour.js

```diff
@@ -78,7 +78,7 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/discuss_public_
         },
         {
             content: "Click on edit",
-            trigger: ".o-mail-Message [title='Edit']",
+            trigger: ".dropdown-menu .dropdown-item[title='Edit'], .o-mail-Message [title='Edit']",
```

Comment:
message prefix was helpful the understand the scope of the button, is it possible to keep it the selector functionally meaningful here? `.o-mail-Message-actions [title='Edit']` like in qunit?